cluster/charts/crossplane-types/templates/clusterrole.yaml

```diff
@@ -17,6 +17,12 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - "*"
```

Comment:
can you help me understand the full set of actions that we need to do on CRDs? the PR description said we need to query them (i.e. GET, LIST), but we also need to create them too when someone defines a new CRD via composition?

Reply:
@jbw976 I can confirm `CREATE` permission is required: https://github.com/crossplane/crossplane/blob/d3e017b94fb24a139739eb71c5f89827d5e85e6a/pkg/controller/apiextensions/definition/reconciler.go#L265

Reply:
It's essentially all operations since we create/update/delete.